web-admin/etc/yvers-parse-version-2

```diff
@@ -215,7 +216,7 @@ function _parseBook( abbrev, chapters, ref_filter = null ) {
     }
   });
   
-  console.log('%s', Inspect( book_json, 8, !'colors' ));
+  console.log('%s', Inspect( book_json, 10, !'colors' ));
```

Comment:
Magic number alert

Reply:
That's just for debugging and should be commented out if not removed.

(It's the depth to which an object/array will be traversed for display)